File: Algorithm/행렬의 덧셈.js
function solution(arr1, arr2) {
    let answer = [];
    for(i=0; i<arr1.length; i++) {
        answer[i] = [];
        for(j=0; j<arr1[i].length; j++) {
            answer[i][j] = arr1[i][j] + arr2[i][j];
        }
    }
    return answer;
}

File: LV1/행렬의 덧셈.go
func solution(arr1 [][]int, arr2 [][]int) [][]int {
	for i := 0; i < len(arr1); i++ {
		for j := 0; j < len(arr1[i]); j++ {
			arr1[i][j] += arr2[i][j]
		}
	}
	return arr1
}
